docs/blocks-storage/migrate-from-chunks-to-blocks.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Migrate Cortex cluster from chunks to blocks"
+linkTitle: "Migrate Cortex cluster from chunks to blocks"
+weight: 5
+slug: migrate-cortex-cluster-from-chunks-to-blocks
+---
+
+This article describes how to migrate existing Cortex cluster from chunks storage to blocks storage,
+and highlight possible issues you may encounter in the process.
+
+_This document replaces the [Cortex proposal](https://cortexmetrics.io/docs/proposals/ingesters-migration/),
```

Comment:
I would update the proposal to mention to not use that proposal as migration reference, but link this new doc page.